examples/aws/storage-efs/README.md

```diff
@@ -0,0 +1,242 @@
+```
+
+This should have installed the EFS CSI Driver into the default kube-system namespace for you. You can check using the following command - 
+```
+kubectl get csidriver
+
+NAME              ATTACHREQUIRED   PODINFOONMOUNT   MODES        AGE
+efs.csi.aws.com   false            false            Persistent   5d17h
+```
+
+
+## 4.0 Create an Instance of the EFS Filesystem
+This section creates a new EFS volume using for your cluster. Please refer to the official [AWS Documents](https://docs.aws.amazon.com/eks/latest/userguide/efs-csi.html) for more details. 
+
+1. Retrieve the VPC ID that your cluster is in and store it in a variable for use in a later step.
+```
+vpc_id=$(aws eks describe-cluster \
+    --name $CLUSTER_NAME\
+    --query "cluster.resourcesVpcConfig.vpcId" \
+    --output text)
+```
+
+2. Retrieve the CIDR range for your cluster's VPC and store it in a variable for use in a later step.
+```
+cidr_range=$(aws ec2 describe-vpcs \
+    --vpc-ids $vpc_id \
+    --query "Vpcs[].CidrBlock" \
+    --output text)
+```
+
+3. Create a security group with an inbound rule that allows inbound NFS traffic for your Amazon EFS mount points.
+    a. Create a security group. 
+    ```
+    security_group_id=$(aws ec2 create-security-group \
+        --group-name MyEfsSecurityGroup \
+        --description "My EFS security group" \
+        --vpc-id $vpc_id \
+        --output text)
+    ```
+
+    b. Create an inbound rule that allows inbound NFS traffic from the CIDR for your cluster's VPC.
+    ```
+    aws ec2 authorize-security-group-ingress \
+        --group-id $security_group_id \
+        --protocol tcp \
+        --port 2049 \
+        --cidr $cidr_range
+    ```
+
+4. Create an Amazon EFS file system for your Amazon EKS cluster.
+```
+file_system_id=$(aws efs create-file-system \
+    --region $CLUSTER_REGION \
+    --performance-mode generalPurpose \
+    --query 'FileSystemId' \
+    --output text)
+```
+
+## 5.0 Create Mount Targets for your cluster
+1. [Optional] If you are re-using an existing EFS Volume, you will first have to delete any old mount targets. You can use the following commands for this - 
+```
+aws efs describe-mount-targets --file-system-id $file_system_id
+aws efs delete-mount-target --mount-target-id <each-id>
+```
+
+2. Determine the IP address of your cluster nodes.
+```
+kubectl get nodes
+```
+
+3. Determine the IDs of the subnets in your VPC and which Availability Zone the subnet is in.
+```
+aws ec2 describe-subnets \
+    --filters "Name=vpc-id,Values=$vpc_id" \
+    --query 'Subnets[*].{SubnetId: SubnetId,AvailabilityZone: AvailabilityZone,CidrBlock: CidrBlock}' \
+    --output table
+```
+
+4. Add mount targets for the subnets that your nodes are in. If are more than 1 nodes in the cluster, you'd run the command once for a subnet in each AZ that you had a node in, replacing subnet-EXAMPLEe2ba886490 with the appropriate subnet ID from the previous command
```

Comment:
I separated them since if you are reusing an existing EFS Volume, you only need to run through this section and not the previous one. I think that is a logical separation and I can make it clearer in the text but ok to merge if everyone prefers.

Reply:
I would encourage it to delete all these and point to AWS docs please. They have better version of it with picture.

Reply:
done.